tools/lsp/server/multiplex.toit

```diff
@@ -39,11 +39,13 @@ class MultiplexConnection:
   buffered-from-compiler_ / BufferedReader
 
   constructor .from-compiler_:
```

Comment:
Alternatively, you could just take the parameter and be explicit about the field assignment in the body. I think that is simpler to reason about, but maybe slightly less compact.

Reply:
I like that more.
thanks and done.